source/sync/data-model/update-schema.txt

```diff
@@ -26,443 +21,518 @@ Update Your Data Model
 
-.. .. figure:: /images/schema_changes.png
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Description of Change
+     - Server-Side Schema 
+     - Client-Side Object Model
+   * - :ref:`Add an object type <schema-update-add-object-type>`
+     - Non-breaking
+     - Non-breaking
+   * - :ref:`Add a property <schema-update-add-required-property>`
+     - Non-breaking
+     - Non-breaking
+   * - :ref:`Add default value on a property <schema-update-add-a-default-value>`
+     - Non-breaking 
+     - Non-breaking
+   * - :ref:`Remove an object type <schema-update-remove-object-type>`
+     - Schema Versioning
+     - Non-breaking
+   * - :ref:`Remove a property <schema-update-remove-property>`
+     - Schema Versioning
+     - Non-breaking
+   * - :ref:`Change whether property is optional or required <schema-update-change-property-status>`
+     - Schema Versioning
+     - Breaking
+   * - :ref:`Rename an object type <schema-update-change-object-type-name>`
+     - Breaking 
+     - Breaking 
+   * - :ref:`Rename a property <schema-update-change-property-name>`
+     - Breaking
+     - Breaking 
+   * - :ref:`Change property type <schema-update-change-property-type-but-keep-name>`
+     - Breaking 
+     - Breaking 
+
+.. tip:: Remap a Name Instead of Renaming
+
+   Several Device SDKs support remapping a property or class name. Depending on
+   your use case, you might choose to remap a name through the client code
+   to avoid having to make a breaking change. For more information, refer to the :ref:`` section 
+
+The following diagram shows the types of changes you can make and the 
+process perform the change:
+
+.. TODO: UPDATE WITH NEW DIAGRAM
+
+.. .. figure:: /images/schema_changes_with_versioning.png
 ..    :alt: Flow chart of schema changes
+..    :lightbox:
+
+Data Model Changes
+------------------ 
+
+The following sections provide more detailed information on how to make
+specific changes to your Device Sync data model.
+
+.. _schema-update-add-object-type:
+
+Add an Object Type
+~~~~~~~~~~~~~~~~~~
+
+You can add an object type to either the server-side schema or the client 
+object model without any additional handling. 
+
+If you want to add an object type to both the server-side schema and the 
+client object model, you can add the object type to the object model, and 
+use :ref:`Development Mode <development-mode>` to let App Services handle the 
+server-side schema updates. Or you can manually add the object type to both 
+the model and the schema.
+
+.. note:: These changes may trigger a resync
+
+   When you add a new object type, we retrieve the documents for the 
+   collection and re-insert them into App Services to get the new values. This
+   is expected behavior, but it does cause a temporary halt to propagating 
+   changes while this process is underway.
+
+.. _schema-update-add-a-default-value:
+
+Add a Default Value on a Property
```

Comment:
Might be nice to link somewhere guiding users on how to do this, since it's a relatively unknown feature

Reply:
I haven't updated these yet - it's next on the list! But the plan is update to reflect any dev mode and schema versioning impacts and to include examples 👍